src/components/Settings/AddUser.tsx

```diff
@@ -136,7 +149,9 @@ export function AddUser({
           setTimeout(() => {
```

Comment:
This seems dangerous to have in the rendering part, but it seems to work so... :shrug: